File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/java/Ejemplo4.java
public class Ejemplo4 {
    public static void main(String[] args) {
        // p1 es una referencia al nuevo objeto Persona
        Persona p1 = new Persona();
        p1.nombre = "Marta";
        p1.edad = 25;
        
        // ptr es otra referencia que apunta al MISMO objeto que p1
        Persona ptr = p1;

        // Se accede a los datos a través de la referencia con el operador "."
        System.out.println("Nombre: " + ptr.nombre);
        System.out.println("Edad: " + ptr.edad);
    }
}

File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/python/ejemplo4.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad

if __name__ == "__main__":
    # p1 es una referencia al nuevo objeto Persona
    p1 = Persona("Marta", 25)
    
    # ptr es otra referencia que apunta al MISMO objeto que p1
    ptr = p1

    # Se accede a los datos a través de la referencia con el operador "."
    print(f"Nombre: {ptr.nombre}")
    print(f"Edad: {ptr.edad}")